### PR TITLE
Adding Sandman Version 1.8.0 Casks

### PR DESCRIPTION
**Pull request by alexanderepstein**
After making all changes to the cask:

- [x]  `brew cask audit --download {{cask_file}}` is error-free.
- [x]  `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

Additionally, if **adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not already refused in [closed issues].
- [x] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/caskroom/homebrew-cask/pulls
[closed issues]: https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed
[the correct repo]: https://github.com/caskroom/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256

### CODE REVIEW

File: Casks/sandman.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask 'sandman' do
+  version '1.8.0'
+  sha256 'faff9c1e068de8a69764a7dff8fa582b688e99855953ca88bc78a667586f5ba4'
+
+  # github.com/alexanderepstein/Sandman was verified as official when first introduced to the cask
+  url "https://github.com/alexanderepstein/Sandman/releases/download/v#{version}/Sandman-#{version}.dmg"
```
Comment (vitorgalvao) [inline Casks/sandman.rb]: Please add an [`appcast`](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md).

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

**Comment by vitorgalvao** [comment]
Thank you for the submission. In the future, please use `x` in checklists (no spaces inside the brackets) and [Github will make them clickable](https://github.com/blog/1375-task-lists-in-gfm-issues-pulls-comments).

**Comment by alexanderepstein** [comment]
@vitorgalvao Sorry I didn't really know how to deal with the checkboxes 😕  but now i do 😄 . I added an appcast and I believe I did it correctly.

**Comment by alexanderepstein** [comment]
@vitorgalvao I fixed the style errors so I think i have all the right stuff for it be merged

**Comment by vitorgalvao** [comment]
Everything looks fine. Thank you and welcome.

**Pull request merged**